stdlib/src/builtin/string.mojo

```diff
@@ -1971,6 +1972,115 @@ struct String(
             )
         return String(buf^)
 
+    @always_inline
+    fn _format[
+        *Ts: Stringable
+    ](
+        inout self,
+        args: VariadicPack[_, _, Stringable, Ts],
+        kwargs: OwnedKwargsDict[String],
```

Comment:
As I mentioned, let's keep the scope limited initially. Variadic keyword-only arguments are very much unstable, so I would prefer we start with just positional substitutions. Then you can extend in a followup patch.

Reply:
Yes, this review can be implemented